src/scanomf.d

```diff
@@ -187,7 +187,8 @@ void scanOmfObjModule(void delegate(const(char)* name, int pickAny) pAddSymbol,
                 parseName(&p, name.ptr);
                 p += (recTyp == PUBDEF) ? 2 : 4; // skip offset
                 parseIdx(&p); // skip type index
-                pAddSymbol(name.ptr, 0);
+                const len = strlen(name.ptr);
+                pAddSymbol(name[0 .. len], 0);
```

Comment:
why the two-step approach with the unnecessary variable instead of simply `p[0 .. strlen(p)]` as you did in line 176?